Sources/SwiftDocC/Model/Rendering/DocumentationContentRenderer.swift

```diff
@@ -365,9 +365,10 @@ public class DocumentationContentRenderer {
         let estimatedTime = (node?.semantic as? Timed)?.durationMinutes.flatMap(formatEstimatedDuration(minutes:))
         
         // Add key information for property lists.
+        // If the symbol overrides the title with a custom name, display the symbol key.
         let propertyListKeyNames = node?.symbol?.plistDetails.map {
             TopicRenderReference.PropertyListKeyNames(
-                titleStyle: ($0.customTitle != nil) ? .useDisplayName : .useRawKey,
+                titleStyle: (titleVariants[.fallback] != nil) ? .useDisplayName : .useRawKey,
```

Comment:
why query on `titleVariants[.fallback]`?

Reply:
Because `.fallback` returns the title variant `defaultValue` value, which is inaccessible directly due to its privacy level. 

The default value of the title variants contains either the title of the render reference node, if this was changed using the DisplayName directive, or nil if it is maintaining the symbol name (not overridden via the DisplayName directive).